### ISSUE TITLE
Error while crafting practice recipe

### ISSUE THREAD

**Issue by Luca-spopo** [CleverRaven/Cataclysm-DDA#61691]
### Describe the bug

The issue happened when I tried to craft the practice recipe for the beginner level leatherworking/tailoring
The error I got was:
```
 DEBUG    : Unable to trim string '' to width 0.  Returning empty string.

 FUNCTION : std::string trim_by_length(const std::string &, int)
 FILE     : src/output.cpp
 LINE     : 275
 VERSION  : 64e17c6
```
The ingredients available to me were thread and sinew
My tailoring level was 1, and I already had the leatherworking proficiency
The error happened just when the menu to ask me to pick ingredients should have showed up

Unfortunately I don't have a save game to help reproduce it.

### Steps to reproduce

I don't know if it is readily reproducible, but

Have thread, sinew and leather patches (and appropriate tools and books) around you
Have tailoring level 1, full proficiency in leatherworking
Attempt to craft the leather practice recipe

### Expected behavior

Menu to ask me to pick ingredients should show up, and after that craft should proceed

### Screenshots

```
 DEBUG    : Unable to trim string '' to width 0.  Returning empty string.

 FUNCTION : std::string trim_by_length(const std::string &, int)
 FILE     : src/output.cpp
 LINE     : 275
 VERSION  : 64e17c6

```

### Versions and configuration

64e17c6
MacOS

**Comment by Luca-spopo** [comment]
Okay I was able to get a save game to repro the issue.
Just load the save, go to the crafting menu > Practice > beginner leather working and try making the item

[New Serbia 15.zip](https://github.com/CleverRaven/Cataclysm-DDA/files/9789153/New.Serbia.15.zip)

**Comment by ZeroInternalReflection** [comment]
I have not specifically checked your save, but this should be fixed by #61626.  Until you update, you should be able to `I`gnore the error message and proceed with crafting without issue.

**Comment by Fris0uman** [comment]
closing as fixed by https://github.com/CleverRaven/Cataclysm-DDA/pull/61626
